contrib/docker/README.md

```diff
@@ -4,24 +4,7 @@ FIXME: this is out-of-date as of
 https://github.com/matrix-org/synapse/issues/5518. Contributions to bring it up
```

Comment:
do you think it is now sufficiently up-to-date to remove this paragraph?

Reply:
yes, i checked this by running docker-compose with almost any variables. As written in the issue, one should go with configuration to homeserver.yaml (however i was surprised opt-in for stats was left here).

Reply:
ok, so please can you remove this paragraph?

Reply:
done, forgot about this.